File: CS/GridControlWithFindPanel/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace GridControlWithFindPanel
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            Random rnd = new Random();
            DataTable table = new DataTable();
            table.Columns.Add("Value1");
            table.Columns.Add("Value2");
            for (int i = 0; i < rnd.Next(100); i++)
            {
                table.Rows.Add(new object[] { "Custom" + rnd.Next(15), "Address" + rnd.Next(20) });
            }
            myGridControl1.DataSource = table;
            checkBox1.Checked = myGridView1.ShowFindPanelOnTop;
        }

        private void checkBox1_CheckedChanged(object sender, EventArgs e)
        {
            myGridView1.ShowFindPanelOnTop = checkBox1.Checked;
        }
    }
}

File: VB/GridControlWithFindPanel/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace GridControlWithFindPanel
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
			Dim rnd As New Random()
			Dim table As New DataTable()
			table.Columns.Add("Value1")
			table.Columns.Add("Value2")
			For i As Integer = 0 To rnd.Next(100) - 1
				table.Rows.Add(New Object() { "Custom" & rnd.Next(15), "Address" & rnd.Next(20) })
			Next i
			myGridControl1.DataSource = table
			checkBox1.Checked = myGridView1.ShowFindPanelOnTop
		End Sub

		Private Sub checkBox1_CheckedChanged(ByVal sender As Object, ByVal e As EventArgs) Handles checkBox1.CheckedChanged
			myGridView1.ShowFindPanelOnTop = checkBox1.Checked
		End Sub
	End Class
End Namespace
